mbtools/html_content_extractor.py

```diff
@@ -0,0 +1,47 @@
+from mbtools import utils
+from pathlib import Path
+import argparse
+
+
+class ContentExtractor:
```

Comment:
Both for code consistency and to be a bit more "pythonic", I think you can avoid this class definition and instead just define `replace_content_tags` and `write_html_files` as a couple of helper functions in this module.